fixtures/soft_assert.py

```diff
@@ -42,24 +42,16 @@
 @pytest.mark.hookwrapper(tryfirst=True)
 def pytest_runtest_protocol(item, nextitem):
     if 'soft_assert' in item.fixturenames:
-        with _soft_assert_cm():
-            yield
-    else:
-        yield
+        _thread_locals.caught_asserts = []
```

Comment:
do we have a plan to restore the artifactor handling of the soft assertions?

Reply:
@RonnyPfannschmidt, please explain